www/api/class/centreon_proxy.class.php

```diff
@@ -10,15 +10,15 @@ public function postCheckConfiguration()
         $proxyAddress = $this->arguments['url'];
         $proxyPort = $this->arguments['port'];
         try {
-            $testUrl = 'http://ci.int.centreon.com:3000/api/pluginpack/pluginpack';
+            $testUrl = 'https://api.imp.centreon.com/api/pluginpack/pluginpack';
             $restHttpLib = new \CentreonRestHttp();
             $restHttpLib->setProxy($proxyAddress, $proxyPort);
             $restHttpLib->call($testUrl);
             $outcome = true;
             $message = _('Connection Successful');
         } catch (\Exception $e) {
             $outcome = false;
-            $message = $e->getMessage();
+            $message = _('Connection failed to imp portal (') . $e->getMessage() . ')';
```

Comment:
Could it be more explicit for users that are not familiar with Centreon ?

Something like "Could not establish connection to Centreon IMP servers (lalala".